### ISSUE TITLE
Mods split across multiple deployers wont be deleted all together

### ISSUE THREAD

**Issue by ZetaDaemon** [limo-app/limo#17]
Just got this working with borderlands 1 today and it's great. I've had to create 3 deployers for `Borderlands`, `Borderlands/WillowGame` and `Borderlands/WillowGame/CookedPC` as mods are shipped in multiple ways and place files across the games folders.
One issue I noticed is that this results in mods essentially being split into multiple mods that cant be deleted all at once, would be nice if these split mods were grouped up together so they could be toggled and deleted all at once

**Comment by limo-app** [comment]
Good idea. I will have to think about the exact implementation, but it might use the same system that would also solve #14.
